### ISSUE TITLE
Support for 3rd party integrations such as Pocket, InstaPaper

### ISSUE THREAD

**Issue by intermx-vignesh** [blenderskool/untab#63]
I just installed this and instantly loved the extension.

But I rarely use bookmarks now and I know many people who use a read later service as a bookmark tool as well.

It would be great to see 3rd party integrations such as InstaPaper and Pocket for the search results, It might be slow but if I want pocket's result, I'm okay to wait.

**Comment by blenderskool** [comment]
Yes, I agree. 3rd party integrations with UnTab would definitely be very useful for users using those platforms! There isn't a set roadmap on this yet, but we should start with fairly simple integrations first, Pocket does sound cool to start with!
